cmd/experimental/kjobctl/hack/update-codegen.sh

```diff
@@ -19,21 +19,23 @@ set -o nounset
 set -o pipefail
 
 GO_CMD=${1:-go}
-KJOBCTL_ROOT=$(realpath $(dirname ${BASH_SOURCE[0]})/..)
+CURRENT_DIR=$(dirname "${BASH_SOURCE[0]}")
+KJOBCTL_ROOT=$(realpath "$CURRENT_DIR/..")
 KJOBCTL_PKG="sigs.k8s.io/kueue/cmd/experimental/kjobctl"
-CODEGEN_PKG=$(cd $TOOLS_DIR && go mod download && $GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
+CODEGEN_PKG=$(cd "$TOOLS_DIR" && go mod download && $GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
 
-cd $(dirname ${BASH_SOURCE[0]})/..
+cd "$CURRENT_DIR/.."
 
+# shellcheck source=/dev/null
```

Comment:
Is it needed? From https://www.shellcheck.net/wiki/SC1090: 
ShellCheck v0.7.2+ will strip a single expansion followed by a slash, e.g. ${var}/util.sh or $(dirname "${BASH_SOURCE[0]}")/util.sh, and treat them as ./util.sh. This allows the use of source-path directives or -P flags to specify the location.

Reply:
Yes, without this directive I get an error:
```
In ./cmd/experimental/kjobctl/hack/update-codegen.sh line 29:
source "${CODEGEN_PKG}/kube_codegen.sh"
       ^-- SC1091 (info): Not following: ./kube_codegen.sh: openBinaryFile: does not exist (No such file or directory)
```

We don't know exactly what the place we will have on `${CODEGEN_PKG}`.

Reply:
Nice, thanks for sharing the error output.